src/utils/AuthService.ts

```diff
@@ -109,113 +111,49 @@ function initializeAuthenticationDev(
             handleSigninSilent(dispatch, userManager);
         }
     }
-    return Promise.resolve(userManager);
+    return userManager;
 }
 
 const accessTokenExpiringNotificationTime = 60; // seconds
 
-function initializeAuthenticationProd(
+export async function initializeAuthenticationProd(
     dispatch: Dispatch<unknown>,
     isSilentRenew: boolean,
-    idpSettings: Promise<Response>,
+    idpSettingsFetcher: IdpSettingsGetter,
```

Comment:
There is a naming mismatch that makes things confusing

Reply:
you mean between "fetcher" and "getter"? that synonyms for me 🤔

Reply:
then choose one :grin: